src/Equations/damaged-elastic/Kernels/Time.cpp

```diff
@@ -703,3 +700,1491 @@ void seissol::kernels::Time::flopsTaylorExpansion(long long& nonZeroFlops,
+    real syz_bm = (2.0 * aB0 + aB1 * xim - aB3 * xim * xim * xim) *
+                  (dofsNMinus[4 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] + epsInityz);
+    real szx_bm = (2.0 * aB0 + aB1 * xim - aB3 * xim * xim * xim) *
+                  (dofsNMinus[5 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] + epsInitzx);
+
+    dofsStressNMinus[0 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        (1 - dofsNMinus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q]) * sxx_sm +
+        dofsNMinus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] * sxx_bm;
+
+    dofsStressNMinus[1 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        (1 - dofsNMinus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q]) * syy_sm +
+        dofsNMinus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] * syy_bm;
+
+    dofsStressNMinus[2 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        (1 - dofsNMinus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q]) * szz_sm +
+        dofsNMinus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] * szz_bm;
+
+    dofsStressNMinus[3 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        (1 - dofsNMinus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q]) * sxy_sm +
+        dofsNMinus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] * sxy_bm;
+
+    dofsStressNMinus[4 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        (1 - dofsNMinus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q]) * syz_sm +
+        dofsNMinus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] * syz_bm;
+
+    dofsStressNMinus[5 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        (1 - dofsNMinus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q]) * szx_sm +
+        dofsNMinus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] * szx_bm;
+
+    dofsStressNPlus[6 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        dofsNPlus[6 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q];
+    dofsStressNPlus[7 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        dofsNPlus[7 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q];
+    dofsStressNPlus[8 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        dofsNPlus[8 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q];
+    dofsStressNPlus[9 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        dofsNPlus[9 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q];
+    dofsStressNPlus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        dofsNPlus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q];
+
+    dofsStressNMinus[6 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        dofsNMinus[6 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q];
+    dofsStressNMinus[7 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        dofsNMinus[7 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q];
+    dofsStressNMinus[8 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        dofsNMinus[8 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q];
+    dofsStressNMinus[9 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        dofsNMinus[9 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q];
+    dofsStressNMinus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q] =
+        dofsNMinus[10 * NUMBER_OF_ALIGNED_BASIS_FUNCTIONS + q];
+  }
```

Comment:
This also seems to be largely a duplicate of the code just above it with `Plus` and `Minus` exchanged. Can you extract this into one function and only call this function twice here.

Reply:
Can't put the calculateEps function in the redefined function as elsewhere, Eps is calculated slightly different